tests/chain_integration/test_delegate_transaction.py

```diff
@@ -0,0 +1,91 @@
+#! pytest
+import pytest
+from tldeploy.identity import MetaTransaction, Identity
+from tldeploy.core import deploy_identity
+
+from relay.blockchain.delegate import Delegate
+from relay.relay import TrustlinesRelay
+
+
+@pytest.fixture(scope='session')
+def delegate_address(web3):
+    return web3.eth.coinbase
+
+
+@pytest.fixture(scope='session')
+def delegate(web3, delegate_address):
+
+    trustlines_relay = TrustlinesRelay()
+    trustlines_relay._load_contracts()
+    identity_contract_abi = trustlines_relay.contracts['Identity']['abi']
```

Comment:
I think this is to big of a dependency, to use the trustlinesRelay object here just to load the contracts. Please just use the `contracts` fixture that can be found in `chain_integration/conftest.py`